plugins/mimecast/plugin.spec.yaml

```diff
@@ -1146,7 +1147,7 @@ actions:
       domain:
         title: Domain
         description: A domain to remove from group. Use either email address or domain
-        example: rapid7.com
+        example: https://example.com
```

Comment:
I think these are supposed to stay rapid7.com or example.com as a domain doesn't include "https://"